obstacle_extractor:
  ros__parameters:
    active: true
    use_scan: true
    use_pcl: false

    use_split_and_merge: true
    circles_from_visibles: true
    discard_converted_segments: true
    transform_coordinates: true
    use_front_half_only: true

    min_group_points: 25

    max_group_distance: 0.1
    distance_proportion: 0.00628
    max_split_distance: 0.2
    max_merge_separation: 0.2
    max_merge_spread: 0.2
    max_circle_radius: 0.6
    radius_enlargement: 0.25

    min_x_limit: -10.0
    max_x_limit: 10.0
    min_y_limit: -10.0
    max_y_limit: 10.0

    frame_id: "base_link"